fobi_custom/override_theme/templates/override_theme/add_form_element_entry_ajax.html

```diff
@@ -0,0 +1,11 @@
+{% extends fobi_theme.form_edit_ajax %}
+
+{% load i18n %}
+
+{% block form_page_title %}
+{% blocktrans with form_element_plugin.name as plugin_name %}<i class="fas fa-xs fa-plus-circle icon-pink"></i>&nbsp; New survey question: <span class="font-italic">{{ plugin_name }}</span>{% endblocktrans %}
+{% endblock form_page_title %}
+
+{% block form_id %}form_element_entry_form{% endblock %}
+
+{% block form_primary_button_text %}{% trans "Add" %}{% endblock %}
```

Comment:
It'd be nice to have a `Cancel` button here that would take you back to the form without adding the field, in addition to `Add`.

Reply:
Took me a minute to figure this out, but added!